test/handlers/kubectl/K8sApplyTLSSecretActionHandlerTestSuite.ts

```diff
@@ -184,7 +184,7 @@ class K8sApplyTLSSecretActionHandlerTestSuite extends K8sBaseHandlerTestSuite {
                     key: key,
                 }
             }, context, snapshot, {})
-        ).to.be.rejected;
+        ).to.be.rejectedWith('Unable to locate cert file for given path');
```

Comment:
Not related to tests, but can we change error message to `Unable to locate cert file with path: ${path}`. Please add the same for key file.